spec/watirspec/elements/button_spec.rb

```diff
@@ -75,6 +75,23 @@
       expect(browser.button(xpath: "//input[@type='button']")).to exist
     end
 
+    it "matches the specific type when locating by type" do
+      aggregate_failures do
```

Comment:
This is a nifty feature, but I think since we don't do this anywhere else in our specs, we shouldn't do it here.